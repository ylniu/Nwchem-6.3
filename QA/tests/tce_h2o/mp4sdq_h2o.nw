#
# Test for CC, CI, MBPT codes in the TCE module
# Reference data obtained by an independent determinantal CC, CI, MBPT code are
#
# LCCD   -0.0505753360
# LCCSD  -0.0508915694
# CCD    -0.0498521356
# CCSD   -0.0501273286
# CCSDT  -0.0502322580
# CCSDTQ -0.0502556616
# QCISD  -0.0501452655
# CISD   -0.0495138383
# CISDT  -0.0496160908
# CISDTQ -0.0502556615
# MBPT2  -0.0358672469
# MBPT3  -0.0098015863
# MBPT4  -0.0030104405
#
# in units of hartree.
#
start h2o

memory  1500 mb

echo

geometry units bohr
O      0.000000000000     0.000000000000    -0.123909374404
H      0.000000000000     1.429936611037     0.983265845431
H      0.000000000000    -1.429936611037     0.983265845431
symmetry c1
end

basis
* library cc-pvdz
end

scf
thresh 1.0e-10
tol2e 1.0e-10
singlet
rhf
end

tce
mbpt4(sdq)
end

task tce energy
