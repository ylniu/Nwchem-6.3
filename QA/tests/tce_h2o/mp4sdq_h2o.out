refund: UserID = d3p852
refund: SLURM Job ID = 1460125
refund: Number of nodes          = 1
refund: Number of cores per node = 1
refund: Number of cores          = 1
refund: Amount of time requested = 0:30
refund: Directory = /home/d3p852/nwchem-5.1.1/QA/tests/tce_h2o
 
Processor list
 
cu01n1
 
All connections between all procs tested: SUCCESS
 argument  1 = /mscf/home/d3p852/nwchem-5.1.1/QA/tests/tce_h2o/mp4sdq_h2o.nw



============================== echo of input deck ==============================
#
# Test for CC, CI, MBPT codes in the TCE module
# Reference data obtained by an independent determinantal CC, CI, MBPT code are
#
# LCCD   -0.0505753360
# LCCSD  -0.0508915694
# CCD    -0.0498521356
# CCSD   -0.0501273286
# CCSDT  -0.0502322580
# CCSDTQ -0.0502556616
# QCISD  -0.0501452655
# CISD   -0.0495138383
# CISDT  -0.0496160908
# CISDTQ -0.0502556615
# MBPT2  -0.0358672469
# MBPT3  -0.0098015863
# MBPT4  -0.0030104405
#
# in units of hartree.
#
start h2o

memory  1500 mb

echo

geometry units bohr
O      0.000000000000     0.000000000000    -0.123909374404
H      0.000000000000     1.429936611037     0.983265845431
H      0.000000000000    -1.429936611037     0.983265845431
symmetry c1
end

basis
* library cc-pvdz
end

scf
thresh 1.0e-10
tol2e 1.0e-10
singlet
rhf
end

tce
mbpt4(sdq)
end

task tce energy
================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.0
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2010
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname      = cu1n1
    program       = /scratch/nwchem
    date          = Tue Nov  2 13:44:00 2010

    compiled      = Mon_Nov_01_15:59:31_2010
    source        = /home/scicons/user/kurt/nwchem-6.0-release-pgf90-final/
    nwchem branch = 6.0
    input         = /mscf/home/d3p852/nwchem-5.1.1/QA/tests/tce_h2o/mp4sdq_h2o.nw
    prefix        = h2o.
    data base     = ./h2o.db
    status        = startup
    nproc         =        1
    time left     =   1768s



           Memory information
           ------------------

    heap     =   49152001 doubles =    375.0 Mbytes
    stack    =   49152001 doubles =    375.0 Mbytes
    global   =   98304000 doubles =    750.0 Mbytes (distinct from heap & stack)
    total    =  196608002 doubles =   1500.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


 Turning off AUTOSYM since
 SYMMETRY directive was detected!


          ------
          auto-z
          ------


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in a.u. (scale by  1.000000000 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 O                    8.0000     0.00000000     0.00000000    -0.22143504
    2 H                    1.0000     0.00000000     1.42993661     0.88574018
    3 H                    1.0000     0.00000000    -1.42993661     0.88574018

      Atomic Mass 
      ----------- 

      O                 15.994910
      H                  1.007825


 Effective nuclear repulsion energy (a.u.)       9.1969343804

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       0.95700
    2 Stretch                  1     3                       0.95700
    3 Bend                     2     1     3               104.50000


            XYZ format geometry
            -------------------
     3
 geometry
 O                     0.00000000     0.00000000    -0.11717839
 H                     0.00000000     0.75668992     0.46871355
 H                     0.00000000    -0.75668992     0.46871355

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |       a.u.
 ------------------------------------------------------------------------------
    2 H                |   1 O                |     1.80847  |     1.80847
    3 H                |   1 O                |     1.80847  |     1.80847
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          2
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    2 H                |   1 O                |   3 H                |   104.50
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          1
 ==============================================================================



  library name resolved from: environment
  library file name is: <
 /mscf/scicons/apps/nwchem-6.0.oct19//src/basis/libraries/>
  


 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                          cc-pvdz                   on all atoms 


                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.17200000E+04  0.000710
  1 S  1.75900000E+03  0.005470
  1 S  4.00800000E+02  0.027837
  1 S  1.13700000E+02  0.104800
  1 S  3.70300000E+01  0.283062
  1 S  1.32700000E+01  0.448719
  1 S  5.02500000E+00  0.270952
  1 S  1.01300000E+00  0.015458

  2 S  1.17200000E+04 -0.000160
  2 S  1.75900000E+03 -0.001263
  2 S  4.00800000E+02 -0.006267
  2 S  1.13700000E+02 -0.025716
  2 S  3.70300000E+01 -0.070924
  2 S  1.32700000E+01 -0.165411
  2 S  5.02500000E+00 -0.116955
  2 S  1.01300000E+00  0.557368

  3 S  3.02300000E-01  1.000000

  4 P  1.77000000E+01  0.043018
  4 P  3.85400000E+00  0.228913
  4 P  1.04600000E+00  0.508728

  5 P  2.75300000E-01  1.000000

  6 D  1.18500000E+00  1.000000

  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.30100000E+01  0.019685
  1 S  1.96200000E+00  0.137977
  1 S  4.44600000E-01  0.478148

  2 S  1.22000000E-01  1.000000

  3 P  7.27000000E-01  1.000000



 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                          cc-pvdz                  6       15   3s2p1d
 H                          cc-pvdz                  3        5   2s1p


                                 NWChem SCF Module
                                 -----------------



  ao basis        = "ao basis"
  functions       =    25
  atoms           =     3
  closed shells   =     5
  open shells     =     0
  charge          =   0.00
  wavefunction    = RHF 
  input vectors   = atomic
  output vectors  = ./h2o.movecs
  use symmetry    = F
  symmetry adapt  = F


 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                          cc-pvdz                  6       15   3s2p1d
 H                          cc-pvdz                  3        5   2s1p



 Forming initial guess at       0.1s


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:         -75.76222910

      Non-variational initial energy
      ------------------------------

 Total energy =     -75.926619
 1-e energy   =    -121.777432
 2-e energy   =      36.653879
 HOMO         =      -0.469523
 LUMO         =       0.090444


 Starting SCF solution at       0.2s



 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          1.000E-10
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-10
 ----------------------------------------------


 #quartets = 3.081D+03 #integrals = 2.937D+04 #direct =  0.0% #cached =100.0%


 Integral file          = ./h2o.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      3        Max. records in file   = 677509
 No. of bits per label  =      8        No. of bits per value  =     64


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1      -75.9923254093  8.37D-01  3.67D-01      0.2
                 2      -76.0248569156  1.80D-01  7.83D-02      0.2
                 3      -76.0271320405  1.48D-02  6.34D-03      0.2
                 4      -76.0271485959  3.53D-05  1.95D-05      0.3
                 5      -76.0271485961  1.07D-09  4.20D-10      0.3
                 6      -76.0271485961  9.28D-12  4.89D-12      0.3


       Final RHF  results 
       ------------------ 

         Total SCF energy =    -76.027148596074
      One-electron energy =   -123.150026011922
      Two-electron energy =     37.925943035405
 Nuclear repulsion energy =      9.196934380443

        Time for solution =      0.1s


             Final eigenvalues
             -----------------

              1      
    1  -20.5518
    2   -1.3371
    3   -0.6997
    4   -0.5670
    5   -0.4935
    6    0.1834
    7    0.2562
    8    0.7893
    9    0.8546
   10    1.1590
   11    1.2000
   12    1.2530
   13    1.4234
   14    1.4762
   15    1.6745

                       ROHF Final Molecular Orbital Analysis
                       -------------------------------------

 Vector    2  Occ=2.000000D+00  E=-1.337094D+00
              MO Center=  4.1D-19, -4.4D-16,  5.4D-02, r^2= 5.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.441073  1 O  s                  3      0.369147  1 O  s          
    16      0.193763  2 H  s                 21      0.193763  3 H  s          

 Vector    3  Occ=2.000000D+00  E=-6.997070D-01
              MO Center= -5.4D-19,  7.3D-16,  1.1D-01, r^2= 7.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.490162  1 O  py                16      0.327990  2 H  s          
    21     -0.327990  3 H  s                  8      0.221760  1 O  py         

 Vector    4  Occ=2.000000D+00  E=-5.669530D-01
              MO Center= -1.5D-17, -6.2D-16, -1.6D-01, r^2= 6.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.546007  1 O  pz                 3     -0.365078  1 O  s          
     9      0.363951  1 O  pz                21      0.206654  3 H  s          
    16      0.206654  2 H  s                  2     -0.153523  1 O  s          

 Vector    5  Occ=2.000000D+00  E=-4.935359D-01
              MO Center=  1.1D-18, -6.2D-17, -9.3D-02, r^2= 6.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.631283  1 O  px                 7      0.495528  1 O  px         

 Vector    6  Occ=0.000000D+00  E= 1.834127D-01
              MO Center=  2.2D-17, -1.6D-13,  6.2D-01, r^2= 3.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      1.068084  1 O  s                 22     -0.838071  3 H  s          
    17     -0.838071  2 H  s                  9      0.341144  1 O  pz         
     6      0.189012  1 O  pz         

 Vector    7  Occ=0.000000D+00  E= 2.561535D-01
              MO Center= -9.7D-17,  1.5D-13,  6.2D-01, r^2= 3.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17      1.444851  2 H  s                 22     -1.444851  3 H  s          
     8     -0.670989  1 O  py                 5     -0.283158  1 O  py         

 Vector    8  Occ=0.000000D+00  E= 7.892712D-01
              MO Center=  2.6D-17, -4.1D-14,  2.5D-01, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    21      0.944451  3 H  s                 16     -0.944451  2 H  s          
    22     -0.685880  3 H  s                 17      0.685880  2 H  s          
     8      0.461886  1 O  py                 5      0.267642  1 O  py         
    14     -0.193602  1 O  dyz               25      0.152972  3 H  pz         
    20     -0.152972  2 H  pz         

 Vector    9  Occ=0.000000D+00  E= 8.545535D-01
              MO Center=  1.3D-17,  3.4D-14,  4.7D-01, r^2= 1.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    16      0.787136  2 H  s                 21      0.787136  3 H  s          
    17     -0.547218  2 H  s                 22     -0.547218  3 H  s          
     6     -0.329459  1 O  pz                 3      0.317417  1 O  s          
    24     -0.296525  3 H  py                19      0.296525  2 H  py         
     2     -0.255423  1 O  s          

 Vector   10  Occ=0.000000D+00  E= 1.158987D+00
              MO Center=  1.7D-17,  2.6D-15, -1.9D-01, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      1.265093  1 O  pz                 6     -0.749404  1 O  pz         
     3      0.565428  1 O  s                 21     -0.535591  3 H  s          
    16     -0.535591  2 H  s                 25      0.244217  3 H  pz         
    20      0.244217  2 H  pz         

 Vector   11  Occ=0.000000D+00  E= 1.200041D+00
              MO Center=  3.7D-18, -2.1D-15, -1.1D-01, r^2= 1.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      1.025618  1 O  px                 4     -0.967736  1 O  px         

 Vector   12  Occ=0.000000D+00  E= 1.253035D+00
              MO Center=  1.3D-16,  8.0D-15, -1.2D-01, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      1.764626  1 O  py                17     -0.826154  2 H  s          
    22      0.826154  3 H  s                  5     -0.734008  1 O  py         
    21      0.379443  3 H  s                 16     -0.379443  2 H  s          
    19      0.302490  2 H  py                24      0.302490  3 H  py         
    25     -0.186952  3 H  pz                20      0.186952  2 H  pz         

 Vector   13  Occ=0.000000D+00  E= 1.423408D+00
              MO Center=  3.2D-18,  7.1D-15,  7.6D-02, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      1.063934  1 O  s                  2     -0.635183  1 O  s          
     9     -0.551951  1 O  pz                20      0.518949  2 H  pz         
    25      0.518949  3 H  pz                24      0.305859  3 H  py         
    19     -0.305859  2 H  py                 1     -0.285593  1 O  s          
    17     -0.274583  2 H  s                 22     -0.274583  3 H  s          

 Vector   14  Occ=0.000000D+00  E= 1.476226D+00
              MO Center= -2.7D-16, -9.7D-15,  4.3D-01, r^2= 1.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    23      0.685639  3 H  px                18     -0.685639  2 H  px         
    11     -0.227326  1 O  dxy        

 Vector   15  Occ=0.000000D+00  E= 1.674473D+00
              MO Center=  3.2D-19,  1.1D-14,  2.9D-01, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    18      0.767151  2 H  px                23      0.767151  3 H  px         
     7     -0.633252  1 O  px                12      0.278483  1 O  dxz        


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =  -0.09752564

 moments of inertia (a.u.)
 ------------------
           6.315769699608           0.000000000000           0.000000000000
           0.000000000000           2.194332428604           0.000000000000
           0.000000000000           0.000000000000           4.121437271004

  Mulliken analysis of the total density
  --------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 O    8     8.30   2.00  0.83  0.82  2.82  1.81  0.02
    2 H    1     0.85   0.69  0.07  0.09
    3 H    1     0.85   0.69  0.07  0.09

       Multipole analysis of the density wrt the origin
       ------------------------------------------------

     L   x y z        total         open         nuclear
     -   - - -        -----         ----         -------
     0   0 0 0      0.000000      0.000000     10.000000

     1   1 0 0      0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000
     1   0 0 1      0.809143      0.000000      0.000000

     2   2 0 0     -5.228889      0.000000      0.000000
     2   1 1 0      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000
     2   0 2 0     -3.072470      0.000000      4.089437
     2   0 1 1      0.000000      0.000000      0.000000
     2   0 0 2     -4.382051      0.000000      1.961339


 Parallel integral file used       1 records with       0 large values

                   NWChem Extensible Many-Electron Theory Module
                   ---------------------------------------------

              ======================================================
                   This portion of the program was automatically
                  generated by a Tensor Contraction Engine (TCE).
                  The development of this portion of the program
                 and TCE was supported by US Department of Energy,
                Office of Science, Office of Basic Energy Science.
                      TCE is a product of Battelle and PNNL.
              Please cite: S.Hirata, J.Phys.Chem.A 107, 9887 (2003).
              ======================================================

            General Information
            -------------------
      Number of processors :     1
         Wavefunction type : Restricted Hartree-Fock
          No. of electrons :    10
           Alpha electrons :     5
            Beta electrons :     5
           No. of orbitals :    50
            Alpha orbitals :    25
             Beta orbitals :    25
        Alpha frozen cores :     0
         Beta frozen cores :     0
     Alpha frozen virtuals :     0
      Beta frozen virtuals :     0
         Spin multiplicity : singlet 
    Number of AO functions :    25
       Number of AO shells :    12
        Use of symmetry is : off
      Symmetry adaption is : off
         Schwarz screening : 0.10D-09

          Correlation Information
          -----------------------
          Calculation type : Fourth-order tensor many-body perturbation theory         wi
   Perturbative correction : none                                                        
            Max iterations :      100
        Residual threshold : 0.10D-06
     T(0) DIIS level shift : 0.00D+00
     L(0) DIIS level shift : 0.00D+00
     T(1) DIIS level shift : 0.00D+00
     L(1) DIIS level shift : 0.00D+00
     T(R) DIIS level shift : 0.00D+00
     T(I) DIIS level shift : 0.00D+00
   CC-T/L Amplitude update :  5-th order DIIS
                I/O scheme : Global Array Library

            Memory Information
            ------------------
          Available GA space size is      98303375 doubles
          Available MA space size is      98302277 doubles

 Warning: unknown model, assuming CCSD-like (tce_tile.F)

 Maximum block size        63 doubles

 tile_dim =     20

 Block   Spin    Irrep     Size     Offset   Alpha
 -------------------------------------------------
   1    alpha     a      5 doubles       0       1
   2    beta      a      5 doubles       5       1
   3    alpha     a     20 doubles      10       3
   4    beta      a     20 doubles      30       3

 Global array virtual files algorithm will be used

 Parallel file system coherency ......... OK

 #quartets = 3.081D+03 #integrals = 2.937D+04 #direct =  0.0% #cached =100.0%


 Integral file          = ./h2o.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      3        Max. records in file   = 677509
 No. of bits per label  =      8        No. of bits per value  =     64


 Fock matrix recomputed
 1-e file size   =              625
 1-e file name   = ./h2o.f1            
 Cpu & wall time / sec            0.1            0.1

 tce_ao2e: fast2e=1
 half-transformed integrals in memory

 2-e (intermediate) file size =         1046875
 2-e (intermediate) file name = ./h2o.v2i           
 Cpu & wall time / sec            0.5            0.5

 tce_mo2e: fast2e=1
 2-e integrals stored in memory

 2-e file size   =           666250
 2-e file name   = ./h2o.v2            
 Cpu & wall time / sec            0.1            0.1
 do_pt =   F
 do_lam_pt =   F
 do_cr_pt =   F
 do_lcr_pt =   F
 do_2t_pt =   F
 T1-number-of-tasks                        1

 t1 file size   =              100
 t1 file name   = ./h2o.t1            
 t1 file handle =       -999
 T2-number-of-boxes                        2

 t2 file size   =            20000
 t2 file name   = ./h2o.t2            
 t2 file handle =       -996

 MBPT(2) iterations
 --------------------------------------------------------
 Iter          Residuum       Correlation     Cpu    Wall
 --------------------------------------------------------
    1   0.0000000000000  -0.2075079616506     0.0     0.0
 --------------------------------------------------------
 Iterations converged
 MBPT(2) correlation energy / hartree =        -0.207507961650617
 MBPT(2) total energy / hartree       =       -76.234656557724780

 MBPT(3) iterations
 --------------------------------------------------------
 Iter          Residuum       Correlation     Cpu    Wall
 --------------------------------------------------------
    1   0.1179864936123   0.0000000000000     0.0     0.0
    2   0.0000000000000  -0.0063706844575     0.0     0.0
 --------------------------------------------------------
 Iterations converged
 MBPT(3) correlation energy / hartree =        -0.006370684457541
 MBPT(3) total energy / hartree       =       -76.241027242182310

 MBPT(4)(SDQ) iterations
 --------------------------------------------------------
 Iter          Residuum       Correlation     Cpu    Wall
 --------------------------------------------------------
    1   0.0313094654389   0.0000000000000     0.1     0.1
    2   0.0000000000000  -0.0023374815836     0.1     0.1
 --------------------------------------------------------
 Iterations converged
 MBPT(4)(SDQ) correlation energy / hartree =        -0.002337481583582
 MBPT(4)(SDQ) total energy / hartree       =       -76.243364723765890

 Parallel integral file used       1 records with       0 large values


 Task  times  cpu:        1.1s     wall:        1.1s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  445      445     9023     1980     1785        0        0        0     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 0.00e+00 0.00e+00
bytes total:             5.34e+07 1.60e+07 1.64e+07 0.00e+00 0.00e+00 0.00e+00
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 13715000 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        17	        31
	current total bytes		         0	         0
	maximum total bytes		   1585392	  22509480
	maximum total K-bytes		      1586	     22510
	maximum total M-bytes		         2	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
      E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski, T. P. Straatsma,
     M. Valiev, H. J. J. van Dam, D. Wang, E. Apra, T. L. Windus, J. Hammond,
     J. Autschbach, P. Nichols, S. Hirata, M. T. Hackler, Y. Zhao, P.-D. Fan,
      R. J. Harrison, M. Dupuis, D. M. A. Smith, K. Glaesemann, J. Nieplocha,
      V. Tipparaju, M. Krishnan, A. Vazquez-Mayagoitia, L. Jensen, M. Swart,
      Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown,
             G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao,
        R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
        B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield,
    X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing, G. Sandrone,
       M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. Wong, Z. Zhang.

 Total times  cpu:        1.1s     wall:        1.2s
